lib/powerpoint.rb

```diff
@@ -3,6 +3,8 @@
 require 'powerpoint/slide/intro'
 require 'powerpoint/slide/textual'
 require 'powerpoint/slide/pictorial'
+require 'powerpoint/slide/text_picture_split'
+require 'powerpoint/slide/picture_description'
```

Comment:
At this point we might consider dynamic inclusion:

`Dir[File.expand_path("../powerpoint/*", __FILE__)].each { |f| require f }`